tests/system_tests/test_artifacts/conftest.py

```diff
@@ -19,3 +24,71 @@ def linked_artifact(wandb_init, user, logged_artifact) -> Artifact:
         run.link_artifact(logged_artifact, "linked-from-portfolio")
 
     return wandb.Api().artifact(f"{user}/other-project/linked-from-portfolio:v0")
+
```

Comment:
are these fixtures useful/used anywhere else except for your examples? seems like you can move these to the relevant file, i don't think it is useful in the conftest file

Reply:
They are only used right now in my tests. My rationale for adding them to the conftest as fixtures was that this will help in writing tests with all various media types going forward.

Reply:
i understand, but given that this is artifacts conftest, i think maybe a better approach is to have a media file (folder) and putting these tests, that actually test media and not artifact there.

Reply:
that makes sense, I will make the change to have a media specifics test folder